### PR TITLE
IA-1953 QA-1197 Enable all tests

### PR DESCRIPTION
**Pull request by Qi77Qi**
https://broadworkbench.atlassian.net/browse/IA-1953
https://broadworkbench.atlassian.net/browse/QA-1197

Root cause of the flakiness is recent privacy change in chrome incognito mode blocks all third parties cookies. We turned this off in https://github.com/broadinstitute/workbench-libs/pull/326

1. consoliate tests in biocondutorSpec a bit so that we're opening less webdrivers at the same time
2. Update the sbt container that we've been using for building automation test container image so that the sbt version available on the image matches up with leonardo.
3. Update scalatest to 3.2.0
4. Looks like `NotebookClusterMonitoringSpec` and `NotebookGCEClusterMonitoringSpec` are failing due to real bug introduced lately. I haven't had time to look yet..but will do after this PR

---
Have you read [CONTRIBUTING.md](https://github.com/DataBiosphere/leonardo/blob/develop/CONTRIBUTING.md) lately? If not, do that first.

I, the developer opening this PR, do solemnly pinky swear that:

- [ ] I've documented my API changes in Swagger

In all cases:

- [ ] Get a thumbsworth of review and PO signoff if necessary
- [ ] Verify all tests go green
- [ ] Squash and merge; Delete your branch after this
- [ ] Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

**Comment by Qi77Qi** [comment]
where is jenkins run? jenkins retest

**Comment by Qi77Qi** [comment]
jenkins retest...fiab-start failure

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM openjdk:8
+FROM oracle/graalvm-ce:20.0.0-java8
```
Comment (rtitle) [inline Dockerfile:1]: Hm… I'm not opposed to it but are we confident in the switch from OpenJDK -> GraalVM?

Comment (Qi77Qi) [reply Dockerfile:1]: welder has been using graalvm....so I think it should be fine...I talked to Albano before about graalvm, he's fine with it too

File: docker/build.sh [hunk 2]
```diff
@@ -75,11 +73,6 @@ while [ "$1" != "" ]; do
             RUN_DOCKER=true
             DOCKER_CMD="$1"
             ;;
-        -dr | --dockerhub-registry)
```
Comment (rtitle) [inline docker/build.sh:78]: Could remove `-ui` above too I guess

File: project/Settings.scala [hunk 3]
```diff
@@ -132,7 +132,7 @@ object Settings {
       * Specify that all tests will be executed in a single external JVM.
       * By default, tests executed in a forked JVM are executed sequentially.
       */
-    Test / fork := true,
+    Test / fork := false,
```
Comment (rtitle) [inline project/Settings.scala:135]: Sounds good to not fork.. does this increase test execution time?

**Review by rtitle — APPROVED** [review]
LGTM, good find on the Chrome fix

File: Dockerfile (hunk 1 above)
Comment (rtitle) [reply Dockerfile:1]: ok

**Comment by xuscrbw** [comment]
Looks good. Includes the update on the assertion for that issue we were running into

File: project/Settings.scala (hunk 3 above)
Comment (Qi77Qi) [reply project/Settings.scala:135]: doesn't seem like it increases execution time much..it's still around 18 min

**Pull request merged**